package.json

```diff
@@ -27,6 +28,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "axe-core": "^4.4.3",
+    "dotenv": "^16.0.3",
```

Comment:
I think we ended up not needing `dotenv`. We can go ahead an remove it. `npm uninstall --save-dev dotenv`

Reply:
https://github.com/sparkbox/accessible-components/pull/34/commits/d2c63ce28db633a9142299a38b395fe45882acee
![delete](https://user-images.githubusercontent.com/69602589/197648327-e1d2b0a4-61d1-49c0-ad56-40975ad352a0.gif)